Bottom: 0a7024b7ae9a9367a5076b9de237cd5b3626804c
Top:    f400d85c19c9be3a3ced7aeff54d661adec2fd84
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-18 16:59:54 +0800

Refresh of tun-reserve-extra-headroom

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 674a8e2..3587689 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -108,6 +108,7 @@ do {								\
 #endif
 
 #define TUN_HEADROOM 256
+/* FIXME */
 #define TUN_RX_PAD (NET_IP_ALIGN + NET_SKB_PAD + TUN_HEADROOM)
 
 /* TUN device flags */
